### ISSUE TITLE
 Furbolg Medicine Pouch effect should not be usable by low level

### ISSUE THREAD

**Issue by Bagiraa** [LightsHope/issues#295]
Item Furbolg Medicine Pouch effect (Restores 100 health every 1 sec for 10 sec) should not be usable by player below level 46.

Proofs : [https://web.archive.org/web/20070701071319/http://www.thottbot.com/i16768](https://web.archive.org/web/20070701071319/http://www.thottbot.com/i16768)
             [https://vanilla-twinhead.twinstar.cz/?issue=10143](https://vanilla-twinhead.twinstar.cz/?issue=10143)

**Comment by slibzx** [comment]
This is not true. The item was changed with the tbc pre-patch so it could not be used by lowlevels. 
Prior to this in vanilla it could.

**Comment by sadfx** [comment]
>  No, another evidence
> 
> From https://web.archive.org/web/20070102103440/http://wow.allakhazam.com/db/item.html?witem=16768

Not exactly evidence as it's not a video where we can see it not working, but I do believe it's correct; it should be equippable, not usable.

**Comment by Gamemechanicwow** [comment]
Spelllevel is set to 50 within spells.dbc in the 1.12 client.

Spelllevel is used to define a min requirement for being a target of a spell. Most common example is when you try to buff a lowlevel player with a highrank spell.
